addons/sale/models/sale_order_line.py

```diff
@@ -347,10 +363,6 @@ def _compute_qty_delivered(self):
         mapping = lines_by_analytic._get_delivered_quantity_by_analytic([('amount', '<=', 0.0)])
         for so_line in lines_by_analytic:
             so_line.qty_delivered = mapping.get(so_line.id or so_line._origin.id, 0.0)
```

Comment:
```suggestion
            so_line.qty_delivered = mapping.get(so_line.id or so_line._origin.id, 0.0)
        (self - lines_by_analytic).qty_delivered = 0.0
```
To have a complete field definition, even if it's not really necessary :).